xlators/mgmt/glusterd/src/glusterd-replace-brick.c

```diff
@@ -405,8 +406,15 @@ glusterd_op_perform_replace_brick(glusterd_volinfo_t *volinfo, char *old_brick,
     if (ret)
         goto out;
 
-    (void)snprintf(new_brickinfo->brick_id, sizeof(new_brickinfo->brick_id),
+    /*(void)snprintf(new_brickinfo->brick_id, sizeof(new_brickinfo->brick_id),
                    "%s", old_brickinfo->brick_id);
+    */
```

Comment:
Why we are keeping this code commented out?
Instead, an small explanation for using memmove, wouldn't be better? For future reference.

Reply:
Yes, Missed it. I will remove this in next commit.